### PR TITLE
Create re-request-review reusable and use it in appropriate places

### LINKED ISSUE #897: Describe how to re-request a review
### What article on docs.github.com is affected?

I think we should update one of the subsections belonging to this main section:

https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests

### What part(s) of the article would you like to see updated?

**Why should the docs be changed?**

I can't find anything in the GitHub docs about re-requesting a review??? I would expect it to be in one of the articles in the *Reviewing changes in pull requests* section, but I can't find it there at all.

**What use cases does it support?**

This should be documented somewhere, so that repository maintainers can link to this section to explain how "re-requesting an review" works.

This also exposes this functionality to anyone who reads the GitHub docs, that way they can "stumble into" this functionality.

**Expected outcome**

There is a section in the GitHub docs that explains that you can click on the refresh icon next to a review, to ping the reviewer for a fresh review.

### Additional information

See the image below for an example of the refresh icon that I'm talking about:

> image

### PR DESCRIPTION
**Pull request by HonkingGoose**
### Why:

Closes #897

### What's being changed:

- Create re-request-review reusable and use it in appropriate places

### Image of reusable:

re-request-review-reusable

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews.md
Comment (HonkingGoose) [inline content/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews.md:52]: I'm not sure where to place this reusable, maybe it should be moved up?

**Review by kobakaxa — APPROVED** [review]

**Review by sky-zadro — APPROVED** [review]
pr:true

**Review by sky-zadro — COMMENTED** [review]
ID:4AEE18F83AFDEB23

**Comment by janiceilene** [comment]
Thanks so much for opening a PR @HonkingGoose! I'll get this triaged for review 🎉

**Comment by HonkingGoose** [comment]
@janiceilene Thanks! I will await the review from one of your teammembers then. :smile:

File: data/reusables/pull_requests/re-request-review.md
Comment (felicitymay) [inline data/reusables/pull_requests/re-request-review.md:2]: It might be worth borrowing the sentence structure from the topic on [pull requests](https://docs-953--docs897.herokuapp.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews#resolving-conversations). This also makes it easy to use some of the other terms that you suggested in the issue for better searchability, and makes sure that people know which tab of the pull request we're talking about.

```suggestion
To request a fresh review from a reviewer, in the sidebar of the **Conversation** tab, click the {% octicon "sync" aria-label="The sync icon" %} icon.
```

**Review by felicitymay — CHANGES_REQUESTED** [review]
👋🏻 Hi @HonkingGoose 

Thanks for identifying and fixing this oversight in the docs. I hadn't spotted the function of that sync icon, so I learnt something new today thanks to you 🙇🏻‍♀️ 

Since this is your first pull request in this repository, I was impressed that you chose to add something that required a resuable and got it working perfectly - this isn't one of the easiest changes to make.

I've made a couple of suggestions, but this should be ready to merge once those are addressed.

File: content/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews.md
Comment (HonkingGoose) [reply content/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews.md:53]: I'll go move this up to line 41 then. :smile:

**Comment by HonkingGoose** [comment]
:wave: Hi @felicitymay,

It's cool that I taught you something new about GitHub. :+1:

> Since this is your first pull request in this repository, I was impressed that you chose to add something that required a resuable and got it working perfectly - this isn't one of the easiest changes to make.

Yeah that took a bit of fiddling to get it working properly. The syntax for the reusable was totally new to me, but it was well documented in the contributing docs. :smile: 

Those suggestions you made were really good, I can see that a competent (and native English language) content writer makes a huge difference in the quality of the text. :heart:

**Review by felicitymay — APPROVED** [review]
Thanks for the changes 😄 

> Yeah that took a bit of fiddling to get it working properly. The syntax for the reusable was totally new to me, but it was well documented in the contributing docs. 😄

It's really good to hear that the docs were clear. Thanks for letting us know.

> Those suggestions you made were really good, I can see that a competent (and native English language) content writer makes a huge difference in the quality of the text. 

I'm always surprised to find out just how many of our contributors are non-native English speakers - some many of you have such great English 🥇 

I'll get this merged. Thanks for your help improving the docs.

**Pull request merged**
